extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -94,6 +94,10 @@ tags: {  # <amp-story>
     name: "title"
     mandatory: true
   }
+  attrs: {
```

Comment:
nit: could you move this above `title` attr? we try to keep the attributes alphabetical by name